distkeras/evaluators.py

```diff
@@ -46,3 +49,33 @@ def evaluate(self, dataframe):
         validated_instances = cleaned.count()
 
         return float(validated_instances) / float(num_instances)
+
+class MseEvaluator(Evaluator):
+    """Computes the MSE accuracy of the prediction based on the label.
+       It adds a new column on the dataframe, default: mse_score
+
+    # Arguments
+        label_col: string. Label column.
+        prediction_col: string. Prediction column.
+        score_col: string. MSE score column
+    """
+
+    def __init__(self, label_col="label", prediction_col="prediction", score_col="mse_score"):
+        # Initialize the parent structure.
+        super(AccuracyEvaluator, self).__init__(label_col, prediction_col)
+        self.score_col = score_col
+
+    def evaluate(self, dataframe):
+        # spark mse udf function 
```

Comment:
Is a UDF really necessary? Can't we just use a lambda function? Or does it have any computational improvements?